src/org.spec.ts

```diff
@@ -255,4 +255,365 @@ describe('org', () => {
       expect(error).toHaveProperty('name', 'MongoNotConnectedError');
     });
   });
+
+  describe('updateOrg', () => {
+    test('Valid data should be allowed and updated', async () => {
```

Comment:
This title suggests we're going to check for two things, but we only check for one (whether the DB was updated). We're not checking whether the result is successful.

However, I think we should do those checks in separate tests, just like we did in the creation test suite above.